### ISSUE TITLE
False positive args[module] in a playbook

### ISSUE THREAD

**Issue by wfleischer** [ansible/ansible-lint#3090]
##### Summary

A task with the `community.docker.docker_container_exec` module falsely fails with a `one of the following is required: argv, command` warning, if placed into a play.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console
ansible-lint 6.13.1 using ansible 2.14.1
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

The following task from the [community.docker.docker_container_exec module example](https://docs.ansible.com/ansible/latest/collections/community/docker/docker_container_exec_module.html#examples) correctly passes the test:

```console
---
- name: Run a simple command (argv)
  community.docker.docker_container_exec:
    container: foo
    argv:
      - /bin/bash
      - "-c"
      - "ls -lah > /dev/stderr"
    chdir: /root
  register: result
```

However, if the same task is put into a play, it fails with a warning `one of the following is required: argv, command`:

```console
---
- name: Test
  hosts: all
  tasks:
    - name: Run a simple command (argv)
      community.docker.docker_container_exec:
        container: foo
        argv:
          - /bin/bash
          - "-c"
          - "ls -lah > /dev/stderr"
        chdir: /root
      register: result
```

##### Desired Behavior

The playbook should pass the test without warnings.
